pkg/flist/cleanup.go

```diff
@@ -127,11 +128,14 @@ func (f *flistModule) cleanupAll() error {
 // Cleaner runs forever, checks the tracker files for filesystem processes
 // that requires cleanup
 func (f *flistModule) Cleaner(ctx context.Context, every time.Duration) {
+	loglimiter := rate.NewLimiter(rate.Every(time.Hour), 1)
```

Comment:
Can this be defined globally instead, with the configured rate, but with a burst of like 10, so each hour we get max of 10 logs ?

Reply:
We could indeed adapt the code so I don't have to do these initialization every time. Maybe a customer logger.
But a burst of 10 means we will have 10 logs the 100 first second, then nothing for the rest of the hour, not sure this is really better.